test/spec.index.js

```diff
@@ -850,6 +850,34 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            it('looks up field label from fields.field-name.options.foo.label if not specified', function () {
```

Comment:
Field names in tests don't make sense. Would something like 'looks up field label from translations when the option is defined as a string' be better?